tests/unit/d.ts

```diff
@@ -99,6 +99,18 @@ registerSuite({
 			assert.lengthOf(dNode.children, 1);
 			assert.isTrue(isWNode(dNode));
 			assert.isFalse(isHNode(dNode));
+		},
+		'create WNode wrapper using symbol label with properties and children'() {
+			const properties: any = { id: 'id', classes: [ 'world' ] };
+			const symbolLabel = Symbol();
+
+			const dNode = w(symbolLabel, properties, [ w(WidgetBase, properties) ]);
+			assert.equal(dNode.type, WNODE);
+			assert.strictEqual(dNode.widgetConstructor, symbolLabel);
+			assert.deepEqual(dNode.properties, { id: 'id', classes: [ 'world' ]});
+			assert.lengthOf(dNode.children, 1);
+			assert.isTrue(isWNode(dNode));
+			assert.isFalse(isHNode(dNode));
```

Comment:
Ok, I was only looking at what test cases got added then paying attention to where they got added.